include/asm/section.hpp

```diff
@@ -73,8 +73,8 @@ void sect_SetLoadSection(
 void sect_EndLoadSection();
 
 Section *sect_GetSymbolSection();
-uint32_t sect_GetSymbolOffset();
-uint32_t sect_GetOutputOffset();
+uint16_t sect_GetSymbolOffset();
```

Comment:
This value still gets used in other contexts where it's 32-bit. Can't that still be allowed here? It's a lot easier to accidentally get a fatal error going above $FFFF bytes than above $FFFF_FFFF.